doc/classes/SurfaceTool.xml

```diff
@@ -124,7 +124,7 @@
 			<param index="0" name="nd_threshold" type="float" />
 			<param index="1" name="target_index_count" type="int" default="3" />
 			<description>
-				Generates a LOD for a given [param nd_threshold] in linear units (square root of quadric error metric), using at most [param target_index_count] indices.
+				Generates an LOD for a given [param nd_threshold] in linear units (square root of quadric error metric), using at most [param target_index_count] indices.
```

Comment:
I would say "lod" as a word, but I guess "el oh dee" is a valid form too.
Only "reference" I could find from a quick search is this shady looking https://www.spellchecker.net/lod

Reply:
Well I'm not sure on that one, might be worth polling on #rendering to see what folks are used to hearing :)

Reply:
Given the current trend for the poll I added back the "LOD" cases, will squash or drop depending

Reply:
I think you can go ahead and squash :)